Add the word custom (#1987)
MicrosoftDocs/powerbi-docs: powerbi-docs/service-report-subscribe.md

* Report page subscriptions are tied to the name of the report page. If you subscribe to a report page and then rename it, you have to re-create your subscription.
* Your organization may configure certain settings in Azure Active Directory that limit the ability to use email subscriptions in Power BI.  These limitations include, but aren't limited to, having multi-factor authentication or IP range restrictions when accessing resources.
* Currently, email subscriptions for reports/dashboards using live connection datasets aren't supported when subscribing users other than yourself unless you are using a paginated report, which will allow you to send the subscription to others using your security context.

> * Email subscriptions don't support most [Power BI custom visuals](developer/visuals/power-bi-custom-visuals.md).  The one exception is those Power BI visuals that have been [certified](developer/visuals/power-bi-custom-visuals-certified.md).  

Reviewer: ```suggestion
* Only default and [certified Power BI visuals](developer/visuals/power-bi-custom-visuals.md) are supported for email subscriptions.  
```
